go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/makerdao/vdb-transformer-utilities
+
+go 1.15
+
+require (
```

Comment:
Not sure if this is intentional but there's a lot of indirect dependencies in here.

Reply:
👍 will run `go mod tidy`